Userland/Libraries/LibWeb/CSS/Default.css

```diff
@@ -243,3 +230,22 @@ summary {
     display: block;
     font-weight: bold;
 }
+
+/* 15.3.1 Hidden elements
+ * https://html.spec.whatwg.org/multipage/rendering.html#hidden-elements
+ */
+/* FIXME: Add `noframes` once frames are implemented. */
+/* FIXME: Add `noembed` once <embed> is implemented. */
+/* FIXME: Add `rp` once <ruby> is implemented. */
+[hidden], area, base, basefont, datalist, head, link, meta, /*noembed,*/
+/*noframes,*/ param, /*rp,*/ script, style, template, title {
+  display: none;
+}
+
+embed[hidden] { display: inline; height: 0; width: 0; }
+
+input[type=hidden i] { display: none !important; }
+
```

Comment:
What does the `i` after `hidden` do? It looks like a typo but I'm always open to learn new web weirdness...

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors

> **`[attr operator value i]`**
> Adding an `i` (or `I`) before the closing bracket causes the value to be compared case-insensitively (for characters within the ASCII range).

Reply:
Your Google foo is better than mine :^)